lib/compat.php

```diff
@@ -182,6 +182,10 @@ function gutenberg_override_reusable_block_post_type_labels() {
  */
 function gutenberg_safe_style_attrs( $attrs ) {
 	$attrs[] = 'object-position';
+	$attrs[] = 'border-top-left-radius';
+	$attrs[] = 'border-top-right-radius';
+	$attrs[] = 'border-bottom-right-radius';
+	$attrs[] = 'border-bottom-left-radius';
```

Comment:
Was this change backported to Core at any point?

Reply:
It appears to me that the [changes were backported](https://github.com/WordPress/wordpress-develop/blob/5eede7436a7938593bdc02b8653aee794b2aee63/src/wp-includes/kses.php#L2317-L2334) and were done so in (https://github.com/WordPress/wordpress-develop/commit/f034bc832eb091079e4130f6a206b4f496766796).

Reply:
Part of my confusion is that the function indicates that it should be removed once the minimum WP version is 5.8, I guess it should have been updated to 5.9 after this change.

Reply:
That was an oversight of mine sorry. This change didn't make it into a release until 11.0 and therefore WP 5.9. 

An additional attribute was added to this filter a few months later (https://github.com/WordPress/gutenberg/pull/36280) that would also make the minimum version requirement 5.9 before it can be removed. 

I've created a tiny PR to fix this comment in https://github.com/WordPress/gutenberg/pull/38359.